File: 260313/정수 사이의 배수/multiplication-between-numbers.py
a, b = map(int, input().split(" "))
sum = 0
cnt = 0

for num in range(a, b+1):
    if num % 5 == 0 or num % 7 == 0:
        sum += num
        cnt += 1

avr = sum / cnt

print(f"{sum} {avr:.1f}")

File: 250529/정수 사이의 배수/multiplication-between-numbers.js
const fs = require('fs');
const [a, b] = fs.readFileSync(0).toString().trim().split(' ').map(Number);

let cnt = 0;
let sum = 0;

for (let i = a; i <= b; i++) {
    if (i % 5 === 0 || i % 7 ===0) {
        sum += i;
        cnt++;
    }
}

const avr = (sum / cnt).toFixed(1);

console.log(sum, avr);